### ISSUE TITLE
[Feature] How to load multiple datasets in a user-defined ratio?

### ISSUE THREAD

**Issue by jzwu48033552** [open-mmlab/mmdetection#9636]
### What's the feature?

Hi,

I want to train a model with two datasets simutaneously. For example, LVIS and CC3m (Conceptual Captions).

It is not a problem that the annotation types of those two datasets are different, I use mask annotations from LVIS and caption annotations from CC3m. My model can handle both inputs in a parallel manner.

The question is: I want to set the sample ratio of the two datasets to 1:4, which means in each training step the dataloader should load 1x images from LVIS and 4x images from CC3m (x depands on batch size).

I have checked the implemented data wrappers and samplers in MMDET but not found a suitable implementation for this.

I understand by using a mixture of RepeatDataset and ConcatDataset in config files can form a dataset that the total number ratio of LVIS and CC3m is close to 1:4 (multiple LVIS for around 6 times), but it still can not control the data ratio in each training step.

I want to know is there any existing implementations to enable the training method above?

If there is not, is there any recommendations how myself implement one?

By the way, it is confusing to me how to use the implemented Samplers in `mmdet/datasets/samplers`. Shall I insert them to config files (how?) or directly instantiate them in the code?

**Comment by Czm369** [comment]
Maybe you can refer to the semi-supervised pipeline 
https://github.com/open-mmlab/mmdetection/blob/3.x/docs/en/user_guides/semi_det.md#configure-semi-supervised-dataloader
